DataFormats/EcalDigi/interface/EcalConstants.h

```diff
@@ -4,6 +4,15 @@
 
 #include "FWCore/Utilities/interface/HostDeviceConstant.h"
 
+// constants used to encode difference between corrected cc time and noncorrected cc time
+// into a uInt8 value to be stored in the 8 bits set aside for the jitter error ( previously unused )
+namespace ecalcctiming {
+  static constexpr const float clockToNS = 25.0;          // Convert clock units to ns
+  static constexpr const float nonCorrectedSlope = 1.2;   // estimates nonCorreted time from corrected time
+  static constexpr const float encodingOffest = 0.32;     // offsets difference in time using clock units
+  static constexpr const float encodingValue = 398.4375;  // encodes time difference into 0 - 255 int range
```

Comment:
For my education, why `encodingValue` that encodes the time difference `t_w_ootc - t_wo_ootc` is fixed and not Era/Condition dependent?

Reply:
By the way, constants defined in a namespace (in contrast to a class) should be defined as `inline constexpr`. `static constexpr` tells the compiler to create a separate copy for each file that includes the header.